README.md

```diff
@@ -1,8 +1,14 @@
+{{< site-region region="us3,eu,gov" >}}
```

Comment:
I'm still concerned about this part, why us1 is excluded 🙂

Reply:
I didn't commit my suggestions yet!